current/ss_wc/SSp99-mapbox/SSp99-mapbox.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
I would expect the files to be named SSp99-mapbox-value.json. Like in EESL where the combination of variables (in this case just plain "value") is referred to..